support/oidc-discovery-provider/source.go

```diff
@@ -0,0 +1,15 @@
+package main
```

Comment:
nit: perhaps this file would be better named `jwks_source.go` ?

Reply:
sgtm